ambry-store/src/main/java/com/github/ambry/store/StoreMetrics.java

```diff
@@ -101,8 +101,11 @@ public class StoreMetrics {
 
   // Compaction related metrics
   public final Counter compactionFixStateCount;
-  public final Meter compactionCopyRateInBytes;
-  public final Counter compactionBytesReclaimedCount;
+  public final Meter compactionCopyRateInBytes;       // per host, physical data are copied during compaction
+  public final Counter compactionBytesReclaimedCount; // per host, the real reclaimed log segment size
+  public final Meter compactionIntervalInMin;           // the interval between two compactions on the same partition.
+  public final Meter compactionPartialLogSegmentCount;  // per host, partial written log segment count
+  public final Meter compactionWastedLogSegmentSpace;   // per host, wasted log segment space
```

Comment:
can we use Gauge for these two metrics, Gauge will show us the value, meter is basically a counter + histogram, which doesn't show us the direct value.

Reply:
Changed back to Gauge. Thanks for the suggestion.